app/views/logs/index.html.erb

```diff
@@ -51,9 +53,11 @@
   </table>
 <% else %>
   <div class="govuk-grid-row">
-    <h1 class="govuk-heading-l govuk-grid-column-full">
+    <h1 class="govuk-heading-l govuk-grid-column-three-quarters">
       <% if params[:username].present? %>
         "<%= params[:username] %>" <strong>isn't reaching the GovWifi service</strong>
+      <% elsif params[:location] %>
+        Traffic from <%= @location_name %> <strong>isn't reaching the GovWifi service</strong>
```

Comment:
Something about this repetition makes me think that the view knows to much about the internals of the search.

I know we don't use presenters but an object that has properties like:
{
search_term: 'foo',
result_count: 39,
results: [...]
}

Would be easy to scale without introducing more branching logic.
Thoughts?

Reply:
Yeah, I like this. Considered doing it, but was worried about how much time I'm spending on this.

If we're happy for me to look at the use case some more, I could do it as part of that.

Reply:
Talked offline, we're happy to move past this for now. Would like to come back to it at some point.